website/community/team.md

```diff
@@ -62,18 +61,30 @@ export function TeamProfileCardCol(props) { return <TeamProfileCard {...props} c
   </TeamProfileCardCol>
 </div>
 
-## Fellows
+## Student Fellows
+
+A handful of students have also worked on Docusaurus as part of their school term/internship and the [Major League Hacking Fellowship program](https://fellowship.mlh.io/), contributing amazing features such as plugin options validation, migration tooling, and a Bootstrap theme.
 
 <div className="row">
   <TeamProfileCardCol name="Anshul Goyal" githubUrl="https://github.com/anshulrgoyal" twitterUrl="https://twitter.com/ar_goyal">
   Fullstack developer who loves to code and try new technologies. In his free time, he contributes to open source, writes blog posts on his <a href="https://anshulgoyal.dev/" target="_blank">website</a> and watches Anime.
   </TeamProfileCardCol>
-  <TeamProfileCardCol name="Tan Teik Jun" githubUrl="https://github.com/teikjun" twitterUrl="https://twitter.com/teik_jun">
-    Open-source enthusiast who aims to become as awesome as the other humans on this page. Working on Docusaurus brought him closer to his goal. 🌱
-  </TeamProfileCardCol>
   <TeamProfileCardCol name="Drew Alexander" githubUrl="https://github.com/drewbi">
     Developer and Creative, trying to gain the skills to build whatever he can think of.
   </TeamProfileCardCol>
+  <TeamProfileCardCol
+    name="Fanny Vieira"
+    githubUrl="https://github.com/fanny"
+    twitterUrl="https://twitter.com/fannyvieiira"
+  >
+    Fanny got started with web development in high school, building a project for the school kitchen. In her free time she loves contributing to Open Source, occasionally writing on <a href="https://dev.to/fannyvieira" target="_blank">her blog</a> about her experiences, cooking, and creating <a href="https://open.spotify.com/user/anotherfanny" target="_blank">Spotify playlists</a>.
+  </TeamProfileCardCol>
+  <TeamProfileCardCol name="Sam Zhou" githubUrl="https://github.com/SamChou19815" twitterUrl="https://twitter.com/SamChou19815">
+    TODO
```

Comment:
@SamChou19815 would you like to provide a short self-description here?

Reply:
Sure.

Sam started programming in 2011 and built his [website](https://developersam.com) in 2015. He is interested in programming languages, dev infra and web development, and has built his own [programming language](https://samlang.developersam.com/) and [mini React](https://github.com/SamChou19815/mini-react).